### ISSUE TITLE
Make wrapper more maintenance-proof with non-breaking refactor

### ISSUE THREAD

**Issue by Dynnammo** [vvaezian/metabase_api_python#37]
Currently, the module is built quite simply with `REST functions` at the beginning of the `metabase-api` and helper functions that helps developers do the most commons actions made on a Metabase instance : create cards, modify them, get informations.

In order to have a more complete wrapper that embrace all the possibilities of the Metabase API and in the same maintaining the simplicity of this wrapper, it could be relevant to do some non-breaking refactoring:
```txt
metabase_api_python # we keep the upper level unchanged
    README.md
    LICENSE
    ...
    metabase_api_python # here's come the changes
        src
            rest_functions.py # contain the REST functions needed by all endpoints
            utils.py # get_item_id, get_item_info etc. go here
            endpoints # containing relevant helpers functions relating to a specific endpoint, 
                cards.py
                dashboards.py
                collections.py
                ... # other endpoints not currently implemented 🔥
        metabase_api.py # keeping Metabase_API class here, that imports all functions from src folder
``` 

**Funded by**
Open Source Politics

**Comment by vvaezian** [comment]
Thanks for the feedback. I'll consider it for future versions.

**Comment by vvaezian** [comment]
Fixed in v0.3.0
